xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -733,6 +734,18 @@ bool CVideoPlayerVideo::ProcessDecoderOutput(double &frametime, double &pts)
     if (m_speed != 0)
       pts += m_picture.iDuration * m_speed / abs(m_speed);
 
+    // ATSC A53 Closed Captions
+    if (CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
+            CSettings::SETTING_SUBTITLES_PARSECAPTIONS) &&
+        m_picture.HasA53SideData())
```

Comment:
can you reverse the content of "if" statement?
so that settings interface is called only when a53 data exists
```suggestion
    if (m_picture.HasA53SideData() && ServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(CSettings::SETTING_SUBTITLES_PARSECAPTIONS))
```